packages/fxa-shared/express/remote-address.js

```diff
@@ -6,7 +6,7 @@
 
 'use strict';
 
-const joi = require('joi');
+const joi = require('@hapi/joi');
```

Comment:
Yes, I do realize that I should be changing from `@hapi/joi` to `joi` and not the other way around.  I'll fix it along with any potential updates from review feedback.